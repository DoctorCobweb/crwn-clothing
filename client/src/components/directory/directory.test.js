import React from 'react';
import { shallow } from 'enzyme';

import { Directory } from './directory.component';

describe('Directory component', () => {
  let wrapper;
  let mockProps;

  beforeEach(()=>{
    mockProps = {
      sections: [{id:0}, {id:1}, {id:2}, {id:3}]
    }
    wrapper = shallow(<Directory {...mockProps} />);
  })

  it('should render Directory component', ()=>{
    expect(wrapper.debug()).toMatchSnapshot();
  });

  it('should render all the sections', ()=>{
    expect(wrapper.find('DirectoryMenuContainer').children().length).toBe(mockProps.sections.length);
  })
});