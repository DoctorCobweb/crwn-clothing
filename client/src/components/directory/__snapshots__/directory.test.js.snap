// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Directory component should render Directory component 1`] = `
"<DirectoryMenuContainer>
  <withRouter(MenuItem) />
  <withRouter(MenuItem) />
  <withRouter(MenuItem) />
  <withRouter(MenuItem) />
</DirectoryMenuContainer>"
`;
